addons/payment_ogone/models/payment_acquirer.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-'8' "-*-"
+
+from hashlib import sha1
+import logging
+import time
+import urlparse
+
+from openerp.addons.payment_ogone.controllers.main import OgoneController
+from openerp import api, fields, models
+from openerp.tools import float_round
+from openerp.tools.float_utils import float_repr
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    @api.model
+    def _get_ogone_urls(self, environment):
+        """ Ogone URLS:
+
+         - standard order: POST address for form-based
+
+        @TDETODO: complete me
+        """
+        return {
+            'ogone_standard_order_url': 'https://secure.ogone.com/ncol/%s/orderstandard_utf8.asp' % (environment,),
+            'ogone_direct_order_url': 'https://secure.ogone.com/ncol/%s/orderdirect_utf8.asp' % (environment,),
+            'ogone_direct_query_url': 'https://secure.ogone.com/ncol/%s/querydirect_utf8.asp' % (environment,),
+            'ogone_afu_agree_url': 'https://secure.ogone.com/ncol/%s/AFU_agree.asp' % (environment,),
+        }
+
+    @api.model
+    def _get_providers(self):
+        providers = super(PaymentAcquirer, self)._get_providers()
+        providers.append(['ogone', 'Ogone'])
+        return providers
+
+    ogone_pspid = fields.Char('PSPID', required_if_provider='ogone')
+    ogone_userid = fields.Char('API User ID', required_if_provider='ogone')
+    ogone_password = fields.Char(
+        'API User Password', required_if_provider='ogone')
+    ogone_shakey_in = fields.Char(
+        'SHA Key IN', required_if_provider='ogone')
+    ogone_shakey_out = fields.Char(
+        'SHA Key OUT', required_if_provider='ogone')
+    ogone_alias_usage = fields.Char('Alias Usage', help="""If you want to use Ogone Aliases,
+                                                                this default Alias Usage will be presented to
+                                                                the customer as the reason you want to
+                                                                keep his payment data""")
+
+    @api.multi
+    def _ogone_generate_shasign(self, inout, values):
```

Comment:
This breaks backward compatibility. You should override same name methods decorated with api.v7 and api.v8 and you should keep the logic in method with decorator api.v8 and you need to call/return the api.v8 decorated version from former.